_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+the talks describing subjects well beyond the expertise of the summary
+author.
+
+### Adjusting Bitcoin's block subsidy
+
+*Research by Anthony (AJ) Towns*
+
+This talk asks whether Bitcoin pays more for security than it needs to
+and then considers options for reducing the amount of subsidy paid in
+the short term as the amount of security increases---still ensuring that
+no more than 21 million bitcoins are paid overall in subsidy, but
+potentially allowing the subsidy to last much longer and also
+potentially reducing the amount of extraneous security (if it exists).
+
+Although the talk was not about a specific proposal, an example proposal
+it evaluated was to reduce the subsidy by 20% every time the network's
+proof of work security doubles (measured by block-creation difficulty).
+
+*[video][vid subsidy], [transcript][tx subsidy], soft fork*
+
+### Forward blocks: on-chain capacity increases without a hard fork
+
+*Research by Mark Friedenbach*
+
+One well-known method for soft forking an increase in the Bitcoin block
+size is extension blocks---a data structure that's invisible to nodes
+that haven't upgraded to the soft fork and so is not subject to their
+historic limits on block size.  By itself, this is an undesirable method
+for increasing block size because preventing old nodes from seeing the
+transactions in the extension block also prevents them from being able
+to enforce any other consensus rules on those transactions---such as
+rules that prevent a malicious user from spending other users' bitcoins
+or from creating more bitcoins than allowed by the 21 million bitcoin
+subsidy schedule.
+
+However, one doesn't need to increase block size to increase the amount
+of data that can be added to the block chain per minute---it's also
+possible to increase capacity by increasing the frequency of blocks
+(reducing the average time between blocks).  A method for gaming
+Bitcoin's difficulty adjustment algorithm---called a time-warp
+attack---is well-known among experts and has been used successfully in
+demonstration attacks against Bitcoin's testnet and real attacks against
+altcoins.  (Note: although Bitcoin is technically vulnerable to this
+attack, it'd be a slow attack that would give the userbase a significant
+amount of time to respond.)  By itself, increasing block frequency is
+also an undesirable method for increasing capacity because shorter block
+intervals increase the effectiveness of miners with large amounts of
+hashrate and so is likely to increase mining centralization.[^freq-pow-waste]
+
+Perhaps disproving the saying that "two wrongs don't make a right," this
+talk describes a novel way of combining extension blocks and the
+time-warp attack to allow both upgraded nodes and old nodes to gain the
+same capacity increase and see all the same transactions for validation
+while simultaneously slightly reducing mining centralization risk.
+Upgraded nodes would validate one or more extension blocks providing
```

Comment:
I think it'd be useful to define the terminology used in the talk, eg "under this proposal, the extension block is called a 'forward block', since it determines which transactions appear in the legacy blocks ahead of those blocks being mined" or similar. Either introduce the terminology here or at the end.